sympy/physics/control/tests/test_lti.py

```diff
@@ -1681,6 +1681,37 @@ def test_StateSpace_functions():
     assert ss3.feedforward_matrix == Matrix([[0, 0], [0, 1]])
 
 
+def test_PIDControl():
+    kp, ki, kd = symbols(['kp','ki','kd'])
+    p1 = PIDController(kp, ki, kd,0, s)
+    # Type Checking
+    assert isinstance(p1, PIDController)
+    assert isinstance(p1, TransferFunction)
+
+    # Properties checking
+    assert p1 == PIDController(kp, ki, kd, 0, s)
+    assert p1.var == s
+    assert p1.num == ki + s*(kd*s + kp)
+    assert p1.den == s
+    assert p1.KP == kp
+    assert p1.KI == ki
+    assert p1.KD == kd
+
+    #Functionality Checking
+    assert p1.doit() == TransferFunction(ki + s*(kd*s + kp), s, s)
+    assert p1.to_expr().expand() == kd*s + ki/s + kp
+
+    #Using PIDController with TransferFunctions
+    tf1 = TransferFunction(s,1,s)
+    #Series combination and Parallel combination with TransferFunction
+    ser1 = Series(p1,tf1)
```

Comment:
```suggestion
    ser1 = Series(p1, tf1)
```

Reply:
Try to make the spacing organized, I see many missed spaces after commas which disturbs overall readability of code.

Reply:
I apologize, sir. I failed to notice it. In the future, I will make sure to thoroughly double-check everything before making any commits.